src/index.js

```diff
@@ -96,27 +97,50 @@ function validateForm() {
     if (!moment(selectedDate.value)._isValid || moment(selectedDate.value).isBefore(moment(Date.now()))) {
         dateError.classList.remove('hidden');
         validated = false;
-    } else {
-        validated = true;
+    // } else {
+    //     validated = true;
     }
-    if (isNaN(selectedDuration.value)) {
+    if (!selectedDuration.value || typeof(+selectedDuration.value) !== 'number') {
         durationError.classList.remove('hidden');
         validated = false;
-    } else {
-        validated = true;
+    // } else {
+    //     validated = true;
     }
-    if (isNaN(selectedTravelers.value)) {
+    if (!selectedTravelers.value || typeof(+selectedTravelers.value) !== 'number') {
         travelersError.classList.remove('hidden');
         validated = false;
-    } else {
-        validated = true;
+    // } else {
+    //     validated = true;
     }
     if (validated) {
         console.log('selectedDate.value', selectedDate.value);
         console.log('selectedDuration.value', selectedDuration.value);
         console.log('selectedTravelers.value', selectedTravelers.value);
         console.log('selectedDestination', selectedDestination.value.split(".")[0]);
+        gatherCompletedTrip(selectedDate.value, selectedDuration.value, selectedTravelers.value, selectedDestination.value.split(".")[0])
+    }
+
+    function gatherCompletedTrip(date, duration, travelers, destinationID) {
+        let foundDestination = allDestinations.find(singleDestination => singleDestination.id === +destinationID);
+        console.log('allDestinations', allDestinations)
+        console.log('foundDestination', foundDestination)
+        let completedTrip = {
+            gatheredDate: date,
+            gatheredDuration: duration,
+            gatheredTravelers: travelers,
+            gatheredDestination: destinationID,
+            gatheredImage: foundDestination.image,
+            gatheredAlt: foundDestination.alt
+        }
+        // domUpdates.displayTripCost(completedTrip);
+        calculateTripCost(completedTrip, foundDestination)
+    }
 
+    function calculateTripCost(desiredTrip, desiredDestination) {
+        let costPerDuration = desiredTrip.gatheredDuration * desiredDestination.estimatedLodgingCostPerDay;
+        let totalPricePerPerson = costPerDuration += desiredDestination.estimatedFlightCostPerPerson;
```

Comment:
```suggestion
        let totalPricePerPerson = costPerDuration + desiredDestination.estimatedFlightCostPerPerson;
```

Can we just add these together instead of reassigning `costPerDuration`?